.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -62,6 +62,13 @@ body:
       placeholder: ex. 1.4.11
     validations:
       required: true
+  - type: input
```

Comment:
[nitpick] I think the arguments should be placed just after `Download SHA256 Hash` and before `Dependencies`. It feels for me more natural to first fill the required fields like `Category` and `Download URL`.